### PR TITLE
ci: add console errors to smoke tests

### PR DESCRIPTION
**Pull request by nickofthyme**
## Summary

Add console errors to smoke tests to help identify the issue. Currently the test just fails and doesn't output the `console.errors`.

### CODE REVIEW

File: browsers/browsers.test.ts [hunk 1]
```diff
@@ -60,6 +60,17 @@ describe('smoke tests', () => {
   });
 
   afterAll(async() => {
+    const entries = await driver.manage().logs().get(logging.Type.BROWSER);
+    const entries2 = await driver.manage().logs().get(logging.Type.CLIENT);
+    console.log('ERRRORS BROWSER', JSON.stringify(entries));
+    console.log('ERRRORS CLIENT', JSON.stringify(entries2));
```
Comment (markov00) [inline browsers/browsers.test.ts:66]: Aaaaaaaaagaaaaaiiiiinnnn thhhhhiiiiss. ccccccccouldddd beeeeee fffffffiiiiiixxxxxeddd :D

Comment (nickofthyme) [reply browsers/browsers.test.ts:66]: lol I'm trying to log the console errors from the webdriver on the browser tests. Any ideas? This doesn't log anything.

Comment (markov00) [reply browsers/browsers.test.ts]: I think the logging API has not been implemented in the IE driver

Comment (markov00) [reply browsers/browsers.test.ts]: and since this will be sunsetted very soon I think we should not care that much

**Pull request closed without merging**